bigchaindb/core.py

```diff
@@ -30,7 +25,7 @@ class Bigchain(object):
     TX_IN_BACKLOG = 'backlog'
     """return if transaction is in backlog"""
 
-    def __init__(self, public_key=None, private_key=None, keyring=[], connection=None):
+    def __init__(self, connection=None, backlog_reassign_delay=None):
```

Comment:
I think, `backlog_reassign_delay` was removed recently. Is it here by mistake?

Reply:
yes, will address. Good catch.